### PR TITLE
TIP-428: move Doctrine readers and writers

### PR DESCRIPTION
**Pull request by jjanvier**
Move Doctrine readers ans writers to a more generic `Database` namespace. (As we don't really use Doctrine inside). The is the twin of the *File* readers/writers.

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | -
| Changelog updated                 | Y
| Review and 2 GTM                  |
| Micro Demo to the PO (Story only) | -
| Migration script                  | - 
| Tech Doc                          | -

### CODE REVIEW

File: UPGRADE-1.6.md [hunk 1]
```diff
@@ -172,4 +172,9 @@ Based on a PIM standard installation, execute the following command in your proj
     find ./src/ -type f -print0 | xargs -0 sed -i 's/implements StandardArrayConverterInterface/implements ArrayConverterInterface/g'
     find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\\Component\\Connector\\ArrayConverter\\Flat/Pim\\Component\\Connector\\ArrayConverter\\FlatToStandard/g'
     find ./src/ -type f -print0 | xargs -0 sed -i 's/pim_connector\.array_converter\.flat\./pim_connector\.array_converter\.flat_to_standard\./g'
+    find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\\Component\\Connector\\Reader\\Doctrine/Pim\\Component\\Connector\\Reader\\Database/g'
+    find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\\Component\\Connector\\Reader\\ProductReader/Pim\\Component\\Connector\\Reader\\Database\\ProductReader/g'
+    find ./src/ -type f -print0 | xargs -0 sed -i 's/pim_connector\.reader\.product/pim_connector\.reader\.database\.product/g'
+    find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\\Component\\Connector\\Writer\\Doctrine/Pim\\Component\\Connector\\Writer\\Database/g'
+    find ./src/ -type f -print0 | xargs -0 sed -i 's/pim_connector\.writer\.database/pim_connector\.writer\.database/g'
```
Comment (skeleton) [inline UPGRADE-1.6.md]: sure ?

Comment (jjanvier) [inline UPGRADE-1.6.md]: good catch

**Comment by skeleton** [comment]
GTM once fixed

**Comment by grena** [comment]
GTM

**Comment by willy-ahva** [comment]
GTM

**Pull request merged**